### ISSUE TITLE
syntax error near unexpected token `elif` in 1.1.15

### ISSUE THREAD

**Issue by drbyte** [laravel/valet#103]
Running any `valet` commands from 1.1.15 gives this error:
```
/usr/local/bin/valet: line 35: syntax error near unexpected token `elif'
/usr/local/bin/valet: line 35: `elif [[ "$1" = "share" ]]'
```

**Comment by adamwathan** [comment]
Should be fixed by 1.1.16. Making such a mess today, sorry everyone 😩

**Comment by drbyte** [comment]
No worries!   
Confirmed fixed in 1.1.16
